spec/factories/job_runs.rb

```diff
@@ -2,8 +2,15 @@
 
 FactoryBot.define do
   factory :job_run do
-    job_type { 'discovery_report' }
     output_location { '/path/to/report' }
     association :batch_context, factory: :batch_context_with_deleted_output_dir
+
+    trait :preassembly do
+      job_type { 'preassembly' }
+    end
+
+    trait :discovery_report do
+      job_type { 'discovery_report' }
+    end
   end
```

Comment:
allow us to actually test different job types for job runs (previously we were always ending up with discovery jobs, even in the preassembly job test)